powerhome/playbook
# Add Matching Decimals Prop to Currency Kit

![React Kit Coverage](https://img.shields.io/badge/React%20Kit%20Coverage-14.31%25-red)This PR makes a new prop available to the currency kit: `decimals`. When this prop is passed the string `"matching"`,  the decimal styling will match the styling of the whole number that proceeds it. This feature is needed by Inifinty Inc as they receive many mocks that use the currency kit, but the mocks have styling that wasn't available. This will increase development speed going forward. 

## React 
![Screen Shot 2022-06-29 at 9 36 25 AM](https://user-images.githubusercontent.com/63466080/176451019-c2311286-1edd-4b73-92de-fd98627fac64.png)

## Rails
![Screen Shot 2022-06-29 at 9 36 39 AM](https://user-images.githubusercontent.com/63466080/176451028-6beb5776-5486-4a07-ba02-98a180324f09.png)

#### Breaking Changes

No; the new optional prop "decimals" for the currency kit will not cause any existing implementations of the currency kit to break. 

#### Runway Ticket URL

None.

#### How to test this

- Verify that the existing playbook Currency kit examples remain unchanged
- Spot check Currency kit implementations in the wild to make sure they are unchanged
- Use the Currency kit and pass the new prop "decimals" with the option "matching":  ensure that the decimals are the same font and size as the whole numbers

#### Checklist:
